components/Skeleton/Skeleton.tsx

```diff
@@ -11,6 +11,14 @@ export const Skeleton = styled('div', {
   position: 'relative',
   overflow: 'hidden',
   borderRadius: '3px',
+  height: 'auto',
+  width: 'auto',
+  '&:not(:empty)': {
+    '& > *': {
+      visibility: 'hidden',
+    },
+    maxWidth: 'fit-content',
+  },
```

Comment:
I'm not sure to get the reason for this part, will we be able to use `Skeleton` around other components? Like
```jsx
<Skeleton>
  <MyComponent />
</Skeleton>
```

Reply:
Nvm, just saw the stories.